src/components/calculator.js

```diff
@@ -6,10 +6,15 @@ const Calculator = () => {
   const [data, setData] = useState({ total: 0, next: '', operation: '' });
 
   // creat handler for button in sfc
-  const HandleClick = (element) => {
+  // const HandleClick = (element) => {
+  //   const math = calculate(data, element.target.textContent);
+  //   setData({ ...data, ...math });
+  // };
```

Comment:
- [ ] It would be better if you remove the unnecessary comments in order to make your codebase more readable and neat.